packages/devextreme/js/__internal/grids/grid_core/header_panel/m_header_panel.ts

```diff
@@ -126,6 +126,13 @@ const members = {
     }
   },
 
+  _isDisabledDefinedByUser(name: string): boolean {
```

Comment:
Let's add the access modifier private to this method.
It looks like it shouldn't be used outside of the m_header_panel :)

```suggestion
  private _isDisabledDefinedByUser(name: string): boolean {
```